docs/current_docs/manuals/developer/snippets/name-overrides/typescript/index.ts

```diff
@@ -0,0 +1,33 @@
+import { dag, Container, object, func, field } from "@dagger.io/dagger"
+
+@object()
+class MyModule {
+  /**
+   * Definition
```

Comment:
I noticed a disparity between the SDKs here. To achieve the output shown in this page for both languages, I needed to document the constructor argument and constructor twice for TypeScript, but only once for Python. Is this intentional @helderco @TomChv ?

Reply:
Sorry, I still didn't understand this. Are the TypeScript and Python snippets equivalent? 
- If yes, then why is the additional inline doc required in the TypeScript code? 
- Or if not, can you please suggest code changes inline to make them equivalent?

Reply:
@TomChv @helderco for help with above.